### PR TITLE
CAL-1295 An information popup should not appear when importing a cale…

### PR DESCRIPTION
**Pull request by nttuyen**
- Bug happen after this ticket KER-318, which store Quartz job into database, require job detail are serializable
- Fix: store the ical input as String instead of InputStream, actually we should not use InputStream of that temporary file in JobDetail, that temporary file may be deleted be for job execution

### CODE REVIEW

File: calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java [hunk 1]
```diff
@@ -735,7 +733,10 @@ public void importCalendarByJob(String username,
     JobDetail job = findImportJob(schedulerService, calendarId);
 
     if(job == null) {
-      job = ImportCalendarJob.getImportICSFileJobDetail(username, calendarId, calendarName, icalInputStream, from, to, isNew);
+      StringWriter writer = new StringWriter();
+      IOUtils.copy(icalInputStream, writer, "UTF-8");
```
Comment (plamarque) [inline calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java]: I think this is wrongly assuming that all imported ics files will use utf8 char set. 
According to the rfc, while it’s the default char set, others may be specified
https://tools.ietf.org/html/rfc5545#section-3.1.4

Comment (nttuyen) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java]: ok @plamarque , let me update PR, i think we don't need to decode String here, only store byte array is enough

File: calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java [hunk 2]
```diff
@@ -16,21 +16,18 @@
...
 import java.util.Arrays;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.TimeZone;
 
 import javax.jcr.Node;
 
+import org.apache.tika.io.IOUtils;
```
Comment (boubaker) [inline calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java]: Please avoid using this Utils Class from non necessary dependency for Calendar. You can use org.apache.commons.compress.utils.IOUtils instead

Comment (nttuyen) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/impl/ICalendarImportExport.java]: @boubaker , i've updated the PR

**Pull request merged**